# HG changeset patch
# User José Luis García Pallero <jgpallero@gmail.com>
# Date 1472910681 -7200
#      Sat Sep 03 15:51:21 2016 +0200
# Node ID 9bc40d4bfe88c6bb641f7b98f7b6defdbc56d15d
# Parent  4c7a66badaa19fdbf483c3fc457e1d7af04fcfe9
es_ES.ts: Update spanish translation of GUI

diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -65,17 +65,17 @@
     <message>
         <location filename="../src/settings-dialog.cc" line="+132"/>
         <source>Difference to the default size</source>
         <translation>Diferencia con el tamaño predeterminado</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
-        <translation>Color de fondo. rosa (255,0,255) representa predeterminado</translation>
+        <translation>Color de fondo; rosa (255,0,255) representa predeterminado</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>b</source>
         <comment>short form for bold</comment>
         <translatorcomment>Abreviatura para caracteres en negrita</translatorcomment>
         <translation>n</translation>
     </message>
@@ -85,17 +85,17 @@
         <comment>short form for italic</comment>
         <translatorcomment>Abreviatura para caracteres en cursiva</translatorcomment>
         <translation>c</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>u</source>
         <comment>short form for underlined</comment>
-        <translatorcomment>Abreviatura para caracteres subrayadas</translatorcomment>
+        <translatorcomment>Abreviatura para caracteres subrayados</translatorcomment>
         <translation>s</translation>
     </message>
 </context>
 <context>
     <name>QTerminal</name>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.h" line="+130"/>
         <source>Copy</source>
@@ -291,17 +291,17 @@
         <location line="+11"/>
         <source>left</source>
         <translation>izquierda</translation>
     </message>
     <message>
         <location line="+5"/>
         <location line="+29"/>
         <source>middle</source>
-        <translation>en medio</translation>
+        <translation>centro</translation>
     </message>
     <message>
         <location line="-24"/>
         <source>right</source>
         <translation>derecha</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -795,17 +795,17 @@ ya se encuentra abierto en el editor</tr
     <message>
         <location line="-117"/>
         <source>Find &amp;Next...</source>
         <translation>&amp;Buscar siguiente...</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Find &amp;Previous...</source>
-        <translation>Buscar &amp;Anterior...</translation>
+        <translation>Buscar &amp;anterior...</translation>
     </message>
     <message>
         <location line="+118"/>
         <source>Show &amp;Indentation Guides</source>
         <translation>Mostrar guías de &amp;sangría</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -815,17 +815,17 @@ ya se encuentra abierto en el editor</tr
     <message>
         <location line="+20"/>
         <source>Zoom &amp;In</source>
         <translation>&amp;Ampliar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom &amp;Out</source>
-        <translation>&amp;Disminuir</translation>
+        <translation>&amp;Reducir</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Normal Size</source>
         <translation>Tamaño &amp;normal</translation>
     </message>
     <message>
         <location line="+32"/>
@@ -845,17 +845,17 @@ ya se encuentra abierto en el editor</tr
     <message>
         <location line="+2"/>
         <source>&amp;Documentation on Keyword</source>
         <translation>&amp;Documentación de palabras clave</translation>
     </message>
     <message>
         <location line="-1171"/>
         <source>The associated file editor tab has disappeared.</source>
-        <translation>La pestaña del editor de texto asociado ha desaparecido.</translation>
+        <translation>La pestaña asociada del editor de texto ha desaparecido.</translation>
     </message>
     <message>
         <location line="+928"/>
         <source>&amp;File</source>
         <translation>&amp;Archivo</translation>
     </message>
     <message>
         <location line="+36"/>
@@ -935,24 +935,24 @@ Warning: The contents in the editor is m
 
 Aviso: ¡el contenido del editor ha sido modificado!</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>It seems that the file
 %1
 has been deleted or renamed. Do you want to save it now?%2</source>
-        <translation>Al parecer el archivo
+        <translation>Parece que el archivo
 %1
 ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
     </message>
     <message>
         <location line="-249"/>
         <source>Line Endings:</source>
-        <translation>Fin de línea:</translation>
+        <translation>Fines de línea:</translation>
     </message>
     <message>
         <location line="-1817"/>
         <source>encoding:</source>
         <translation>Codificación:</translation>
     </message>
     <message>
         <location line="+239"/>
@@ -960,29 +960,29 @@ ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
         <translatorcomment>Condición de &quot;breakpoint&quot;</translatorcomment>
         <translation>Condición de punto de interrupción</translation>
     </message>
     <message>
         <location line="+89"/>
         <source>Cannot add breakpoint to modified file.
 Save and add breakpoint, or canel?</source>
         <translation>No es posible agregar punto de interrupción al archivo modificado.
-¿Guardar y agregar punto de interrupción, o cancelar?</translation>
+¿Desea guardar y agregar punto de interrupción o cancelar?</translation>
     </message>
     <message>
         <location line="+1322"/>
         <source>Debug or Save</source>
         <translation>Depurar programa o guardar</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>This file is currently being executed.
 Quit debugging and save?</source>
         <translation>Este archivo esta siendo ejecutado.
-¿Interrumpir depuración y guardar?</translation>
+¿Desea interrumpir la depuración y guardar?</translation>
     </message>
     <message>
         <location line="+175"/>
         <source>File Encoding:</source>
         <translation>Codificación de archivo:</translation>
     </message>
     <message>
         <location line="+28"/>
@@ -1017,18 +1017,18 @@ is not a valid identifier.
 
 If you keep this filename, you will not be able to
 call your script using its name as an Octave command.
 
 Do you want to choose another name?</source>
         <translation>&quot;%1&quot;
 no es un identificador válido.
 
-Si mantiene ese nombre de archivo, no le será posible 
-llamar su script usando su nombre como comando de octave.
+Si mantiene ese nombre de archivo no le será posible
+llamar a su script usando su nombre como comando de Octave.
 
 ¿Desea escoger otro nombre?</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>The current editor contents can not be encoded
 with the selected codec %1.
 Using it will result in data loss!
@@ -1037,17 +1037,17 @@ Do you want to chose another codec?</sou
         <translation>El contenido actual del editor no puede ser codificado
 con el codec seleccionado %1.
 ¡Usando dicho codec provocará pérdida de datos!
 ¿Desea seleccionar otro codec?</translation>
     </message>
     <message>
         <location line="+83"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
-        <translation>Al parecer el archivo &apos;%1&apos; ha sido modificado por otra aplicación. ¿Desea recargarlo?</translation>
+        <translation>Parece que el archivo &apos;%1&apos; ha sido modificado por otra aplicación. ¿Desea recargarlo?</translation>
     </message>
 </context>
 <context>
     <name>files_dock_widget</name>
     <message>
         <location filename="../src/files-dock-widget.cc" line="+69"/>
         <source>File Browser</source>
         <translation>Explorador de archivos</translation>
@@ -1065,33 +1065,33 @@ con el codec seleccionado %1.
     <message>
         <location line="+4"/>
         <source>Actions on current directory</source>
         <translation>Acciones en el directorio actual</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Set Browser Directory...</source>
-        <translation>Asignar directorio de navegador...</translation>
+        <translation>Asignar directorio del navegador...</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+373"/>
         <source>Find Files...</source>
         <translation>Buscar archivos...</translation>
     </message>
     <message>
         <location line="-26"/>
         <source>Open in System File Explorer</source>
         <translation>Abrir en explorador de archivos del sistema</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Open in Text Editor</source>
-        <translation>Abrir en editor de texto</translation>
+        <translation>Abrir en el editor de texto</translation>
     </message>
     <message>
         <location line="+154"/>
         <source>Rename file/directory:
 </source>
         <translation>Renombrar archivo/directorio:
 </translation>
     </message>
@@ -1265,17 +1265,17 @@ con el codec seleccionado %1.
     <message>
         <location line="+1"/>
         <source>Can not delete a directory that is not empty</source>
         <translation>No se puede eliminar un directorio que no esté vacío</translation>
     </message>
     <message>
         <location line="+145"/>
         <source>Set directory of file browser</source>
-        <translation>Seleccionar directorio de explorador de archivos</translation>
+        <translation>Seleccionar el directorio del explorador de archivos</translation>
     </message>
     <message>
         <location line="+29"/>
         <source>Create File</source>
         <translation>Crear archivo</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1356,17 +1356,17 @@ a:link { text-decoration: underline; col
         <translation>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;Para más información acerca de Octave:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visite &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt; (se abre en una ventana externa del navegador)&lt;/li&gt;
-&lt;li&gt;Obtenga la documentación en linea en formato &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt; o como &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;documento&lt;/span&gt;&lt;/a&gt; pdf(se abre en una ventana externa del navegador)&lt;/li&gt;
+&lt;li&gt;Obtenga la documentación en línea en formato &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt; o como &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;documento&lt;/span&gt;&lt;/a&gt; pdf (se abre en una ventana externa del navegador)&lt;/li&gt;
 &lt;li&gt;Abra el navegador de documentación de la GUI de Octave desde el menú de ayuda&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;
 
 
 </translation>
     </message>
 </context>
@@ -1800,17 +1800,17 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
     <message>
         <location line="+12"/>
         <source>Octave Release Notes</source>
         <translation>Notas de la versión de Octave</translation>
     </message>
     <message>
         <location line="+192"/>
         <source>Octave Community News</source>
-        <translation>Noticas de la comunidad Octave</translation>
+        <translation>Noticas de la comunidad de Octave</translation>
     </message>
     <message>
         <location line="+206"/>
         <source>Are you sure you want to exit Octave?</source>
         <translation>¿Está seguro de que desea salir de Octave?</translation>
     </message>
     <message>
         <location line="+916"/>
@@ -2052,17 +2052,18 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
     <message>
         <location line="-30"/>
         <source>&amp;Help</source>
         <translation>&amp;Ayuda</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Report Bug</source>
-        <translation>Informar de fallo</translation>
+        <translatorcomment>Añado entre paréntesis la palabra bug porque ya está totalmente implantada en el mundo castellanoparlante</translatorcomment>
+        <translation>Informar de fallo (bug)</translation>
     </message>
 </context>
 <context>
     <name>news_reader</name>
     <message>
         <location line="-1492"/>
         <source>&lt;html&gt;
 &lt;body&gt;
@@ -2108,17 +2109,17 @@ or enable web connections for news in Oc
 &lt;body&gt;
 &lt;p&gt;
 La conexión a la web para mostrar las últimas novedades sobre Octave ha sido desactivada.
 &lt;/p&gt;
 &lt;p&gt;
 Para ver las últimas novedades, vaya a
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;
 cuando tenga conexión a la web (el enlace se abre en un navegador externo)
-o active la conexión a la web para ver las novedades en el diálogo de configuración de red de Octave
+o actívela para ver las novedades en el diálogo de configuración de red de Octave
 &lt;/p&gt;
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; Los desarrolladores de Octave,
 </translation>
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
@@ -2186,27 +2187,27 @@ no existe. ¿Desea crearlo?</translation>
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
         <location line="+254"/>
         <source>The file %1 does not exist in the load path.  To run or debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translation>El archivo %1 no existe en las rutas de búsqueda de funciones (path). Para ejecutar o depurar la función que está editando, tiene que cambiar al directorio %2 o agregar ese directorio a las rutas de búsqueda de funciones (path).</translation>
+        <translation>El archivo %1 no existe en las rutas de búsqueda de funciones (path). Para ejecutar o depurar la función que está editando tiene que cambiar al directorio %2 o agregarlo a las rutas de búsqueda de funciones (path).</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path. To run or debug the function you are editing, change to the directory %2.</source>
-        <translation>Existe un archivo con el mismo nombre que %1 en la ruta de búsqueda de funciones (path). Para ejecutar o depurar la función que se está editando, cambie al directorio %2.</translation>
+        <translation>Existe un archivo con el mismo nombre que %1 en la ruta de búsqueda de funciones (path). Para ejecutar o depurar la función que se está editando cambie al directorio %2.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
-        <translation>Cambiar directorio o agregar directorioa la ruta de búsqueda de funciones (path)</translation>
+        <translation>Cambiar directorio o agregar directorio a la ruta de búsqueda de funciones (path)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory</source>
         <translation>Cambiar directorio</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -2411,17 +2412,17 @@ La interfaz gráfica de Octave se cerrará.</translation>
     <message>
         <location line="-173"/>
         <source>Font</source>
         <translation>Tipo de fuente</translation>
     </message>
     <message>
         <location line="-1007"/>
         <source>Show line numbers</source>
-        <translation>Mostrar numeros de línea</translation>
+        <translation>Mostrar números de línea</translation>
     </message>
     <message>
         <location line="+81"/>
         <source>Highlight current line</source>
         <translation>Resaltar línea actual</translation>
     </message>
     <message>
         <location line="+13"/>
@@ -2446,17 +2447,17 @@ La interfaz gráfica de Octave se cerrará.</translation>
     <message>
         <location line="+124"/>
         <source>Create nonexistent files without prompting</source>
         <translation>Crear archivos no existentes sin consultar</translation>
     </message>
     <message>
         <location line="-441"/>
         <source>command line (%f=file, %l=line):</source>
-        <translation>línea de comando (%f=archivo, %l=linea):</translation>
+        <translation>línea de comandos (%f=archivo, %l=linea):</translation>
     </message>
     <message>
         <location line="+613"/>
         <source>Terminal</source>
         <translation>Terminal</translation>
     </message>
     <message>
         <location line="+51"/>
@@ -2712,17 +2713,17 @@ La interfaz gráfica de Octave se cerrará.</translation>
         <location line="+3"/>
         <location line="+80"/>
         <source>Default</source>
         <translation>Valor predeterminado/por defecto</translation>
     </message>
     <message>
         <location line="-71"/>
         <source>Edit a shortcut by double-clicking in a cell</source>
-        <translation>Los Accesos Directos se pueden editar haciendo doble clic en la celda correspondiente</translation>
+        <translation>Los accesos directos se pueden editar haciendo doble clic en la celda correspondiente</translation>
     </message>
     <message>
         <location line="+76"/>
         <source>Actual</source>
         <translation>Actual</translation>
     </message>
     <message>
         <location line="+17"/>
@@ -2767,22 +2768,22 @@ La interfaz gráfica de Octave se cerrará.</translation>
     <message>
         <location line="+184"/>
         <source>Shortcuts</source>
         <translation>Accesos Directos de teclado</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Select this option to prevent conflicts with readline shortcuts</source>
-        <translation>Seleccione esta opción para evitar conflictos con los Accesos Directos de la línea de comandos</translation>
+        <translation>Seleccione esta opción para evitar conflictos con los accesos directos de la línea de comandos</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Disable global shortcuts when terminal window has focus</source>
-        <translation>Deshabilitar los Accesos Directos globales cuando la ventana del terminal se enfoque </translation>
+        <translation>Deshabilitar los accesos directos globales cuando la ventana del terminal se enfoque </translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Export</source>
         <translation>Exportar</translation>
     </message>
     <message>
         <location line="-10"/>
@@ -2894,17 +2895,17 @@ La interfaz gráfica de Octave se cerrará.</translation>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;When the Octave GUI starts, it will check the Octave web site for current news and information about the Octave community.
 The check will happen at most once each day and news will only be displayed if there is something new since the last time you viewed the news.&lt;/p&gt;
 &lt;p&gt;You may also view the news by selecting the &quot;Community News&quot; item in the &quot;Help&quot; menu in the GUI, or by visiting
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;Cuando la GUI se inicia, chequeará el sitio web de Octave para obtener novedades e información acerca de la comunidad.
-Esta verificación se hará una vez cada día como máximo, y las noticias serán desplegadas únicamente si hay algo nuevo desde la última vez que se vieron.&lt;/p&gt;
+Esta verificación se hará una vez cada día como máximo y las noticias serán desplegadas únicamente si hay algo nuevo desde la última vez que se vieron.&lt;/p&gt;
 &lt;p&gt;También puede revisar las noticias seleccionando &quot;Noticias de la Comunidad&quot; en el menú de ayuda en la GUI o visitando
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+26"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
@@ -3179,22 +3180,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Find and Replace</source>
         <translation>Buscar y reemplazar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find Next</source>
-        <translation>Buscar Siguiente</translation>
+        <translation>Buscar siguiente</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find Previous</source>
-        <translation>Buscar Anterior</translation>
+        <translation>Buscar anterior</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Word</source>
         <translation>Eliminar hasta el inicio de palabra</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3311,89 +3312,89 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+4"/>
         <source>Show Line Numbers</source>
         <translation>Mostrar numeros de línea</translation>
     </message>
     <message>
         <location line="+48"/>
         <source>Switch to Left Tab</source>
-        <translation>Cambiar a Pestaña Izquierda</translation>
+        <translation>Cambiar a pestaña izquierda</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Switch to Right Tab</source>
-        <translation>Cambiar a Pestaña Derecha</translation>
+        <translation>Cambiar a pestaña derecha</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move Tab Left</source>
-        <translation>Mover Pestaña Izquierda</translation>
+        <translation>Mover pestaña izquierda</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move Tab Right</source>
-        <translation>Mover Pestaña Derecha</translation>
+        <translation>Mover pestaña derecha</translation>
     </message>
     <message>
         <location line="+44"/>
         <source>Global</source>
         <translation>Global</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>Tabs</source>
         <translation>Pestañas</translation>
     </message>
     <message>
         <location line="+98"/>
         <source>Enter new Shortcut</source>
-        <translation>Introducir nuevo Acceso Directo</translation>
+        <translation>Introducir nuevo acceso directo</translation>
     </message>
     <message>
         <location line="+143"/>
         <source>Overwriting Shortcuts</source>
-        <translation>Sobrescribir Accesos Directos</translation>
+        <translation>Sobrescribir accesos directos</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>You are about to overwrite all shortcuts.
 Would you like to save the current shortcut set or cancel the action?</source>
-        <translation>Está a punto de sobrescribir todos los Accesos Directos.
-¿Le gustaría salvar el conjunto actual de Accesos Directos o cancelar esta acción?</translation>
+        <translation>Está a punto de sobreescribir todos los accesos directos.
+¿Le gustaría salvar el conjunto actual de accesos directos o cancelar esta acción?</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Don&apos;t save</source>
-        <translation>No salvar</translation>
+        <translation>No guardar</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Import shortcuts from file ...</source>
-        <translation>Importar Accesos Directos de archivo ...</translation>
+        <translation>Importar accesos directos de archivo ...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Export shortcuts into file ...</source>
-        <translation>Exportar Accesos Directos al archivo ...</translation>
+        <translation>Exportar accesos directos al archivo ...</translation>
     </message>
     <message>
         <location line="-571"/>
         <source>Step</source>
         <translation>Siguiente instrucción</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Reset Default Window Layout</source>
         <translation>Restablecer el esquema de ventana predeterminado</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Report Bug</source>
-        <translation>Informar de fallo</translation>
+        <translation>Informar de fallo (bug)</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Close Other Files</source>
         <translation>Cerrar otros archivos</translation>
     </message>
     <message>
         <location line="+30"/>
@@ -3433,27 +3434,27 @@ Would you like to save the current short
     <message>
         <location line="+2"/>
         <source>Show Long Line Marker</source>
         <translation>Mostrar marcador de longitud de línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Toolbar</source>
-        <translation>Mostrar Barra de Herramientas</translation>
+        <translation>Mostrar barra de herramientas</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Statusbar</source>
         <translation>Mostrar barra de estado</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Horizontal Scrollbar</source>
-        <translation>Mostrar Barra de Desplazamiento Horizontal</translation>
+        <translation>Mostrar barra de desplazamiento horizontal</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom In</source>
         <translation>Aumentar</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -3548,66 +3549,66 @@ Would you like to save the current short
     <message>
         <location line="+4"/>
         <source>Run</source>
         <translation>Ejecutar</translation>
     </message>
     <message>
         <location line="+106"/>
         <source>Apply the desired shortcut or click on the right button to reset the shortcut to its default.</source>
-        <translation>Aplique el Acceso Directo que desee o haga clic en el botón derecho para restablecer el Acceso Directo a su valor predeterminado.</translation>
+        <translation>Aplique el acceso directo que desee o haga clic en el botón derecho para restablecer el acceso directo a su valor predeterminado.</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Enter shortcut directly by performing it</source>
-        <translation>Introduzca un Acceso Directo.ejecutándolo directamente </translation>
+        <translation>Introduzca un acceso directo ejecutándolo directamente</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Actual shortcut</source>
-        <translation>Acceso Directo actual</translation>
+        <translation>Acceso directo actual</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Default shortcut</source>
-        <translation>Acceso Directo por defecto</translation>
+        <translation>Acceso directo por defecto</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Set to default</source>
         <translation>Establecer como por defecto</translation>
     </message>
     <message>
         <location line="+45"/>
         <source>Double Shortcut</source>
-        <translation>Acceso Directo doble</translation>
+        <translation>Acceso directo doble</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The chosen shortcut
   &quot;%1&quot;
 is already used for the action
   &quot;%2&quot;.
 Do you want to use the shortcut anyhow removing it from the previous action?</source>
-        <translation>El Acceso Directo seleccionado
+        <translation>El acceso directo seleccionado
   &quot;%1&quot;
 ya se utiliza para la acción
   &quot;%2&quot;.
-¿Desea utilizar el Acceso Directo de cualquier modo suprimiendo la acción previamente asignada?</translation>
+¿Desea utilizar el acceso directo de cualquier modo suprimiendo la acción previamente asignada?</translation>
     </message>
     <message>
         <location line="+113"/>
         <location line="+5"/>
         <source>Octave Shortcut Files (*.osc);;All Files (*)</source>
-        <translation>Archivo de Acceso Directo de Octave (*.osc);;Todos los archivos (*)</translation>
+        <translation>Archivo de acceso directo de Octave (*.osc);;Todos los archivos (*)</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Failed to open %1 as octave shortcut file</source>
-        <translation>No se ha podido abrir %1 como archivo de Accesos Directos de Octave</translation>
+        <translation>No se ha podido abrir %1 como archivo de accesos directos de Octave</translation>
     </message>
 </context>
 <context>
     <name>terminal_dock_widget</name>
     <message>
         <location filename="../src/terminal-dock-widget.cc" line="+38"/>
         <source>Command Window</source>
         <translation>Ventana de comandos</translation>
@@ -3718,17 +3719,17 @@ ya se utiliza para la acción
     <message>
         <location line="+3"/>
         <source>Rename</source>
         <translation>Renombrar</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Only top-level symbols may be renamed</source>
-        <translation>Sólo símbolos de nivel superior pueden ser renombrados</translation>
+        <translation>Sólo los símbolos de nivel superior pueden ser renombrados</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>Hide filter</source>
         <translation>Ocultar filtro</translation>
     </message>
     <message>
         <location line="+3"/>
